community-operators/openshift-nfd-operator/4.6/nfd.v4.6.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,208 @@
+    mediatype: image/png
+  install:
+    spec:
+      clusterPermissions:
+      - rules:
+        - apiGroups:
+          - rbac.authorization.k8s.io
+          resources:
+          - roles
+          - rolebindings
+          - clusterroles
+          - clusterrolebindings
+          verbs:
+          - '*'
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          - services
+          - endpoints
+          - persistentvolumeclaims
+          - events
+          - configmaps
+          - secrets
+          - serviceaccounts
+          - nodes
+          verbs:
+          - '*'
+        - apiGroups:
+          - ""
+          resources:
+          - namespaces
+          verbs:
+          - '*'
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          - daemonsets
+          - replicasets
+          - statefulsets
+          verbs:
+          - '*'
+        - apiGroups:
+          - monitoring.coreos.com
+          resources:
+          - servicemonitors
+          verbs:
+          - get
+          - create
+        - apiGroups:
+          - nfd.openshift.io
+          resources:
+          - '*'
+          verbs:
+          - '*'
+        - apiGroups:
+          - policy
+          resourceNames:
+          - nfd-worker
+          resources:
+          - podsecuritypolicies
+          verbs:
+          - use
+        - apiGroups:
+          - security.openshift.io
+          resources:
+          - securitycontextconstraints
+          verbs:
+          - '*'
+        serviceAccountName: nfd-operator
+      deployments:
+      - name: nfd-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: nfd-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: nfd-operator
+            spec:
+              containers:
+              - command:
+                - cluster-nfd-operator
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: cluster-nfd-operator
+                - name: NODE_FEATURE_DISCOVERY_IMAGE
+                  value: quay.io/openshift/origin-node-feature-discovery:latest
+                image: quay.io/openshift/origin-cluster-nfd-operator:latest
```

Comment:
Is there a possibility to use release-tagged manager images with release-tagged deployment artifacts? I think this could help prevent further conflicts if the API changes in any way, without having to build deprecated/changed API compatibility into every release forever.

Reply:
yes it is, I will update the PR